prow/config.yaml

```diff
@@ -10529,6 +10529,26 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- cron: '1 13 * * 2,4,6' # Run at 5:01PST (13:01 UTC) on even days
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-scale-coredns
+  tags:
+  - "perfDashPrefix: gce-2kNodes-master"
```

Comment:
Please remove these tags here and below (we don't want these to be relevant to perfDash).